docs/trace_collection.md

```diff
@@ -18,35 +18,84 @@ Send [traces][1] to Datadog from your Android applications with [Datadog's `dd-s
 
 2. Initialize the library with your application context, tracking consent, and the [Datadog client token][4] and Application ID generated when you create a new RUM application in the Datadog UI (see [Getting Started with Android RUM Collection][7] for more information). For security reasons, you must use a client token: you cannot use [Datadog API keys][5] to configure the `dd-sdk-android` library as they would be exposed client-side in the Android application APK byte code. For more information about setting up a client token, see the [client token documentation][4]:
 
-   {{< tabs >}}
-   {{% tab "US" %}}
+#### US
+
+{{< tabs >}}
+{{% tab "Kotlin" %}}
    ```kotlin
-    class SampleApplication : Application() {
-        override fun onCreate() {
-            super.onCreate()
-            val configuration = Configuration.Builder(tracesEnabled = true, ...).build()
-            val credentials = Credentials(<CLIENT_TOKEN>, <ENV_NAME>, <APP_VARIANT_NAME>, <APPLICATION_ID>)
-            Datadog.initialize(this, credentials, configuration, trackingConsent)
-        }
-    }
+      class SampleApplication : Application() {
+
+   override fun onCreate() {
+      super.onCreate()
+      val configuration = Configuration.Builder(
+              tracesEnabled = true
+      )
+              .build()
+      val credentials = Credentials(<CLIENT_TOKEN>, <ENV_NAME>, <APP_VARIANT_NAME>, <APPLICATION_ID>)
+      Datadog.initialize(this, credentials, configuration, trackingConsent)
+   }
+}
    ```
-   {{% /tab %}}
-   {{% tab "EU" %}}
+{{% /tab %}}
+{{% tab "Java" %}}
+   ```java
+      public class SampleApplication extends Application {
```

Comment:
```suggestion
public class SampleApplication extends Application {
```
out-dent the following lines too, to line 54. Throughout, I think the code blocks can be out-dented to the 1st column within the code block. We can see how it looks in preview.